### DISCUSSION TITLE
Support for literals/delimiters surrounds?

### DISCUSSION THREAD

**Discussion by gldtn** [chrisgrieser/nvim-various-textobjs#110] [Q&A] [answered]
Is it possible to target quotes/brackets and curly braces individually? If not, can you implement it or is outside of this plugin purpose?

Sometimes I want to substitute those and seems like a nice addition for this plugin if it doesn't already support it. I went through the list and did not see it mentioned.

I know there is a ton of surround plugins out there, but would be nice to have a single plugin that deals with text objects.

Thanks for the plugin BTW!

**Comment by chrisgrieser** [comment, marked as answer]
Thanks!

Yeah, Surrounds are out of scope for this plugin, since they use a very different internal logic. I'd suggest using one of those surround plugins for that indeed
